set

```diff
@@ -0,0 +1,5 @@
+set KBC_TOKEN=572-153951-cDrt6Wo8Fa73ptN35ZrTYxEHpPGI3Zk5GCAP6vcY
+set KBC_CONFIG_ID=467550007
```

Comment:
we were using `.env` for these files.  Anyhow, pretty sure this shouldn't be here